flink-runtime/src/test/java/org/apache/flink/runtime/state/StateBackendMigrationTestBase.java

```diff
@@ -395,7 +399,7 @@ public void testKeyedMapStateStateMigration() throws Exception {
                         new TestType.V2TestTypeSerializer()));
     }
 
-    @Test
+    @TestTemplate
     public void testKeyedMapStateSerializerReconfiguration() throws Exception {
```

Comment:
```suggestion
    void testKeyedMapStateSerializerReconfiguration() throws Exception {
```
could be package private